Dockerfile

```diff
@@ -2,8 +2,11 @@ FROM python:3.6
 
 RUN mkdir -p /src/ngsi-timeseries-api
 
-COPY requirements.txt /src/ngsi-timeseries-api/requirements.txt
+COPY Pipfile /src/ngsi-timeseries-api/Pipfile
+COPY Pipfile.lock /src/ngsi-timeseries-api/Pipfile.lock
 
+RUN pip install pipenv
```

Comment:
Let's move this line above the two copy lines and then feel free to merge!

Reply:
done!